### PR TITLE
Add schema translations test for null answer label

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?
This adds test for schema .pot file translations and extraction of translatable items when there is empty("null") label present in answers.

### How to review 
Run `make test`, run extract and translate of checkbox schema with empty label.

### CODE REVIEW

**Comment by petechd** [comment]
[P1: Checkbox default text to be removed - NI](https://trello.com/c/91PIVgpv/4266-p1-checkbox-default-text-to-be-removed-ni)

File: tests/test_survey_schema_translation.py [hunk 1]
```diff
@@ -504,3 +504,32 @@ def test_locale_on_load_ulster_scots():
...
+    catalog = Catalog(locale=Locale("cy"))
+    catalog.add(
+        "Rugby", "Rygbi",
+    )
+    schema_translation.catalog = catalog
+    schema = SurveySchema(
+        {
+            "question": {
+                "answers": [
+                    {"label": None, "options": [{"label": "Rugby", "value": "Rugby"},],}
```
Comment (ajmaddaford) [inline tests/test_survey_schema_translation.py:521]: I think this formatting is probably because these were originally on multiple lines. 
```suggestion
                    {"label": None, "options": [{"label": "Rugby", "value": "Rugby"}]}
```

File: tests/test_survey_schema_translation.py [hunk 2]
```diff
@@ -504,3 +504,32 @@ def test_locale_on_load_ulster_scots():
...
+                ],
+            }
+        }
+    )
+    translated = schema.translate(schema_translation)
+    expected = {
+        "question": {
+            "answers": [
+                {"label": None, "options": [{"label": "Rygbi", "value": "Rugby"},],}
```
Comment (ajmaddaford) [inline tests/test_survey_schema_translation.py:530]:
```suggestion
                {"label": None, "options": [{"label": "Rygbi", "value": "Rugby"}]}
```

**Review by ajmaddaford — APPROVED** [review]

File: tests/test_survey_schema_translation.py [hunk 3]
```diff
@@ -504,3 +504,32 @@ def test_locale_on_load_ulster_scots():
...
+    catalog = Catalog(locale=Locale("cy"))
+    catalog.add(
+        "Rugby", "Rygbi",
+    )
+    schema_translation.catalog = catalog
+    schema = SurveySchema(
+        {
+            "question": {
+                "answers": [
+                    {"label": None, "options": [{"label": "Rugby", "value": "Rugby"}]}
```
Comment (LJBabbage) [inline tests/test_survey_schema_translation.py:521]: I am slightly confused, why is it None here, it was null in the schema repo?

Comment (petechd) [reply tests/test_survey_schema_translation.py:521]: it's because when python parses json files it changes it to `None` as there is no `null` in python, same in runner templated we used `if label == None`.

Comment (LJBabbage) [reply tests/test_survey_schema_translation.py:521]: Ugly

**Review by LJBabbage — APPROVED** [review]
